nmrfx-core/src/main/java/org/nmrfx/chemistry/io/PDBFile.java

```diff
@@ -89,18 +89,9 @@ public static String getReslibDir() {
         }
     }
 
-    public MoleculeBase read(String fileName)
+    public MoleculeBase read(String fileName, boolean strictMode)
```

Comment:
This is one of three methods in this file called from the MoleculeMenuActions readMolecule method which is matched with the Molecule menu options that are a little unclear, should a Jira issue be created for improving this? @brucejohnson  https://github.com/nanalysis/nmrfx/blob/7275cb109929e49aa2c2bfe587d25031dc3ec26c/nmrfx-analyst-gui/src/main/java/org/nmrfx/analyst/gui/molecule/MoleculeMenuActions.java#L44
https://github.com/nanalysis/nmrfx/blob/7275cb109929e49aa2c2bfe587d25031dc3ec26c/nmrfx-analyst-gui/src/main/java/org/nmrfx/analyst/gui/molecule/MoleculeMenuActions.java#L147